spec/index.html

```diff
@@ -371,26 +408,66 @@ <h3>Triple Terms and Reification</h3>
       </a>
       <figcaption id="fig-asserted-triple-term-alt">
         An <a>RDF graph</a> containing a <a>triple annotation</a>, where the <a>triple term</a> of a <a>reifying triple</a> corresponds to an <a>asserted triple</a>.
-        The diagram represents the proposition as a fact using the asserted triple, meaning that the relationship holds.
+<!-- 
+Slightly rewriting the below statement.        
+-->
+<!-- original:
+The diagram represents the proposition as a fact using the asserted triple, meaning that the relationship holds.
+-->
+        Due to the asserted triple, the diagram represents the proposition as a fact, meaning that the relationship holds.
       </figcaption>
     </figure>
 
+<!-- I found the sentence to be quite difficult to read - here is an attempt to simplify. -->
+<!-- original:
+    <p>When the triple term of a <a>reifying triple</a> also [=appears=] in the
+      same graph as an <a>asserted triple</a>, the subset of triples that share
+      the same <a>reifier</a> as subject is called a <dfn>triple annotation</dfn>.</p>
+-->
+
+    <p>Here, the triple term of a <a>reifying triple</a> also [=appears=] in the
+      same graph as an <a>asserted triple</a>. In this case, 
+      the subset of triples including the <a>reifier</a> as subject&mdash;for instance, providing metadata as in these examples&mdash;
+      is called a <dfn>triple annotation</dfn>.
```

Comment:
Defining terminology is better placed in the text before the diagram and its label.

Reply:
This was an attempt to reduce the complexity of the section - it takes quite some text to introduce the terminology

> When the triple term of a <a>reifying triple</a> also [=appears=] in the
      same graph as an <a>asserted triple</a>, the subset of triples that share
      the same <a>reifier</a> as subject is called a <dfn>triple annotation</dfn>.

To understand all this without having seen the example first was a lot of effort for me.

Reply:
Fair enough and I moved it back to above the figure.